vast_pipeline/management/commands/ingestimages.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from django.core.management.base import BaseCommand
```

Comment:
```suggestion
from django.core.management.base import BaseCommand, CommandError
```

see suggestion below